server/types.graphql

```diff
@@ -95,4 +95,16 @@ type User @model {
 
   createdAt: DateTime!
   updatedAt: DateTime!
+
+  consentForPersonalData: Consent @relation(name: "UserConsent")
+}
+
+type Consent @model {
+  id: ID! @isUnique
+  user: [User!]! @relation(name: "UserConsent")
```

Comment:
"[User!]!" indique une liste d'utilisateurs. Ici on est sur une relation 1-1, donc à remplacer par "User"

Reply:
Right. Actually I'm going to change the relationship to 1 user - n consents in order to have one row per consent. It's more flexible that way if we need to add some.

Reply:
What's the benefit compared to adding an attribute to the Consent model ?

Reply:
The schema does not change.